google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/QueryRequestInfoTest.java

```diff
@@ -171,8 +174,19 @@ public void testToPb() {
   @Test
   public void equalTo() {
     compareQueryRequestInfo(
-        new QueryRequestInfo(QUERY_JOB_CONFIGURATION_SUPPORTED), REQUEST_INFO_SUPPORTED);
-    compareQueryRequestInfo(new QueryRequestInfo(QUERY_JOB_CONFIGURATION), REQUEST_INFO);
+        new QueryRequestInfo(QUERY_JOB_CONFIGURATION_SUPPORTED, false), REQUEST_INFO_SUPPORTED);
+    compareQueryRequestInfo(new QueryRequestInfo(QUERY_JOB_CONFIGURATION, false), REQUEST_INFO);
+  }
+
+  @Test
+  public void testInt64Timestamp() {
+    QueryRequestInfo qri = new QueryRequestInfo(QUERY_JOB_CONFIGURATION, false);
```

Comment:
Nit.
variable names qri/qr/qri2/qr2 departs from the style norm of the rest of the repo.

Reply:
Good point, variable names have been updated to better match the repo